lib/geoservices/queryable.rb

```diff
@@ -14,7 +14,15 @@ def query(layer_idx=0, options={})
     # returns nil if the layer does not have the Capability: Query
     def count(layer_idx=0, options={})
       return nil unless self.metadata["capabilities"] =~ /Query/
-      query(layer_idx, options.merge(:returnCountOnly => true))
+      # Current Version is not returned at lower than 10.1
+      unless self.metadata['currentVersion'].nil?
+        query(layer_idx, options.merge(:returnCountOnly => true))
+      else
+        # Return count only is not supported by versions lower than 10.1
+        # So request all the object ids and count them up
+        response = query(layer_idx, options.merge(:where => '1=1', :returnIdsOnly => true))
```

Comment:
I don't believe that's true. This version of server does not give a max record count. But this one does (2000).
See more than 2000 records:
http://services.arcgis.com/bkrWlSKcjUDFDtgw/arcgis/rest/services/Crime_Incidents_DC_2014/FeatureServer/0/query?where=1=1&returnIdsOnly=true&f=json